og_ui/src/BundleFormAlter.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\og_ui\BundleFormAlter.
+ */
+
+
+namespace Drupal\og_ui;
+
+
```

Comment:
I think we should have one line above and below the namespace as we have in other files.